cirq/ops/raw_types.py

```diff
@@ -153,6 +153,38 @@ def _json_dict_(self):
         return protocols.obj_to_dict_helper(self, ['qubit', 'dimension'])
 
 
+class Tag:
+    """A wrapper around string that can be user to differentiate gates.
+
+    This can be instantiated using a string value or sub-classed to
+    differentiate or tag specific instances of gates.
+
+    For instance,
+    cirq.X().add_tag(Tag('do_not_touch')
```

Comment:
missing close paren

Reply:
Done.  Also changed this so it renders correctly as example code.  There were a bunch of other issues in this code as well.